.github/workflows/R-CMD-check.yaml

```diff
@@ -26,10 +21,21 @@ jobs:
     env:
       PTAXSIM_DB_PATH: ${{ github.workspace }}/ptaxsim.db
       R_KEEP_PKG_SOURCE: yes
+
+    # Required for OIDC access to S3
+    permissions:
+      id-token: write
+      contents: read
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Prepare PTAXSIM database
+        uses: ./.github/actions/prepare-ptaxsim
+        with:
+          ASSUMED_ROLE: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
```

Comment:
This is a hacky way to pass a secret to a composite action. Needed until this issue https://github.com/actions/toolkit/issues/1168 is resolved. It still properly masks the secret in the output logs.